src/components/Subscribe/index.js

```diff
@@ -0,0 +1,124 @@
+import React from "react"
+import { useTheme } from "emotion-theming"
+import { FormattedMessage } from "gatsby-plugin-intl"
+
+import {
+  fontStyle,
+  mdtImage,
+  maiImage,
+  socialIcon,
+  subscribeContainer,
+  subscribe,
+  subscribeTitle,
+  emailAddressContainer,
+  emailAddress,
+  allRightsReserved,
+  poweredBy,
+  poweredByContainer,
+  otherPages,
+  contentContainer,
+  contentLeftContainer,
+  logosContainer,
+  contentRightContainer,
+  greenArrow,
+} from "./style"
+
+import RightArrowIcon from "@/assets/right-arrow.svg"
+import mailogo from "@/images/mai-logo-2.svg"
+import mdtLogo from "@/images/mdt-logo.svg"
+import facebookIcon from "@/images/btn-facebook.svg"
+import linkedinIcon from "@/images/btn-linkedin.svg"
+import mediumIcon from "@/images/btn-medium.svg"
+import twitterIcon from "@/images/btn-twitter.svg"
+import wechatIcon from "@/images/btn-wechat.svg"
+import weiboIcon from "@/images/btn-weibo.svg"
+
+const SocialIcon = ({ last, src, alt }) => (
+  <img css={socialIcon(last)} src={src} alt={alt} />
+)
+
+const OtherPages = props => (
+  <p css={theme => [fontStyle(theme), otherPages(theme)]} {...props}></p>
+)
+const GreenRightArrow = () => {
+  const theme = useTheme()
+  return (
+    <RightArrowIcon
+      css={greenArrow}
+      fill={theme.colors.greens.dark}
+      width={21}
+      height={29}
+    />
+  )
+}
+
+const Subscribe = ({ ...props }) => (
+  <div {...props}>
+    <div css={subscribeContainer}>
+      <p css={theme => [fontStyle(theme), subscribeTitle]}>
+        <FormattedMessage
+          id="subscribeToOurNewsLetter"
+          defaultMessage={`<span>SUBSCRIBE</span> to our newsletter`}
+          values={{
+            span: str => <span css={subscribe}>{str}</span>,
+          }}
+        />
+      </p>
+    </div>
+    <div css={contentContainer}>
+      <div css={contentLeftContainer}>
+        <div css={emailAddressContainer}>
+          <p css={emailAddress}>
+            <FormattedMessage
+              id="emailAddress"
+              defaultMessage="Email Address"
+            />
+          </p>
+          <GreenRightArrow />
```

Comment:
I find that this arrow is different from others (the dots needs to be larger for this one). Use another image for it.

Reply:
changed.